src/pages/product/Product.css

```diff
@@ -0,0 +1,88 @@
+.wrapper_product {
+  display: grid;
+  grid-template-columns: 24rem 1fr;
+  grid-template-areas: "aside main";
+  padding-top: 6rem;
+  user-select: none;
+}
+aside {
+  grid-area: aside;
+  position: fixed;
+  z-index: var(--index-999);
+}
+main {
+  grid-area: main;
+}
+
+/*product card */
+
+.wrapper_card_product {
+  padding: 1rem;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  gap: 2rem;
+  min-height: 100vh;
+}
+.card_product {
+  margin: 0;
+  width: 30rem;
+  transform: scale(0.9);
+}
+.card_name_product {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 30rem;
+}
+.card_img_product {
+  padding: 1rem 0;
+  width: 20rem;
+  height: 20rem;
+  object-fit: cover;
+}
+.wishlist_icon {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  z-index: var(--index-999);
+  font-size: 2.4rem;
+  cursor: pointer;
+  border: none;
+  background-color: var(--white);
+}
+.emptyWishlist {
+  color: #cccccc;
+}
+.filledWishlist {
+  color: #f22426;
```

Comment:
try not to hard code colors. Use variables instead

Reply:
Made Changes! Thanks!